packages/nx-go/src/executors/lint/schema.json

```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "https://json-schema.org/schema",
   "version": 2,
-  "title": "Lint executor",
+  "title": "@nx-go/nx-go:lint",
```

Comment:
**question:** Is this the best practice for Nx plugins?

Just want to understand the reasoning behind the title change.

Reply:
I don't think there's a good practice for this field. But in the official executors, it seems more like a real title and not an identifier like here. I'll go back to the original value, thanks for the feedback.